docs/sources/enterprise/auditing.md

```diff
@@ -14,41 +14,42 @@ Auditing allows you to track important changes to your Grafana instance. By defa
 ## Audit logs
 
 Audit logs are JSON objects representing user actions like:
+
 - Modifications ro resources such as dashboards and data sources.
 - A user failing to log in.
 
 ### Format
 
 Audit logs contain the following fields. The fields followed by **\*** are always available, the others depends on the type of action logged.
```

Comment:
🤯 @ `**\***`.

Markdown always reminds me why I kind of like HTML or LaTeX after all 😛